docs/faststore/docs/managing-performance/overview.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: "Managing performance"
+---
+
+# Performance 
+
+In the ecommerce business, the performance of your website direclty impacts the shoppers' experience. It may affect **sales conversion rate**, **user session time**, and other relevant metrics. Every millisecond counts and influences not only the shopper's decision-making process but also your store's website **rank in search engine results**. Therefore, it is crucial to ensure that your website is as fast and performant as possible, achieving **high scores in [Lighthouse](https://developer.chrome.com/docs/lighthouse/overview) audits**.
```

Comment:
```suggestion
In the ecommerce business, the performance of your website directly impacts the shoppers' experience. It may affect **sales conversion rate**, **user session time**, and other relevant metrics. Every millisecond counts and influences not only the shopper's decision-making process but also your store's website **rank in search engine results**. Therefore, it is crucial to ensure that your website is as fast and performant as possible, achieving **high scores in [Lighthouse](https://developer.chrome.com/docs/lighthouse/overview) audits**.
```